spring-boot-mvc-2/pom.xml

```diff
@@ -1,68 +1,73 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
We generally want to avoid changing only whitespaces like changing the indent from tabs to spaces or vice versa on an existing file. Please restore this to its original indentation, and then reapply any other changes you have made, if any.

Reply:
done